src/DependencyInjection/Configuration.php

```diff
@@ -37,6 +37,55 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode
             ->addDefaultsIfNotSet()
             ->children()
+                ->arrayNode('pdf_rendering')
+                ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('default_orientation')->defaultValue('Portrait')->end()
+                        ->arrayNode('available_orientations')
```

Comment:
No need for this I think

Reply:
We need at some point to allow extension of available parameters imo

Reply:
Okay, but it won't be a breaking change to add this by then. Right now I can't think of other orientations than portrait and landscape

Reply:
I would prefer to keep it, for consistency reasons (we have that for format as well), and also because it does not add complexity at all, and may allow any dev to simply disable one orientation (w/e their reasons). What do you think ?

Reply:
Okay 👍